client/inbox-panel/index.js

```diff
@@ -184,7 +185,12 @@ const InboxPanel = ( { showHeader = true } ) => {
 
 			return {
 				notes: getNotes( INBOX_QUERY ).map( ( note ) => {
-					note.content = truncateRenderableHTML( note.content, 320 );
+					if ( note.locale === 'en_US' ) {
```

Comment:
Since we're supporting is general English, this needs to support other English variations as well. According to [translate.wordpress.org](https://translate.wordpress.org/), the official English locales are:

- en_US
- en_AU
- en_CA
- en_GB
- en_ZA

![image](https://user-images.githubusercontent.com/3747241/145322785-8392e80f-5e0c-4345-9307-86dcd7a3435d.png)

Reply:
Thank you so much for catching this! 👍  Updated it in https://github.com/woocommerce/woocommerce-admin/pull/8022/commits/c370bd53633a9452ad75434fc6536526d4637665

Reply:
LGTM! 🚢